src/uct/tcp/sockcm/sockcm_iface.c

```diff
@@ -267,6 +275,7 @@ static UCS_CLASS_INIT_FUNC(uct_sockcm_iface_t, uct_md_h md, uct_worker_h worker,
         self->cb_flags         = params->mode.sockaddr.cb_flags;
         self->conn_request_cb  = params->mode.sockaddr.conn_request_cb;
         self->conn_request_arg = params->mode.sockaddr.conn_request_arg;
+        self->sockaddr         = param_sockaddr;
```

Comment:
why are you saving sockaddr on the iface? where is it used?

Reply:
This is for the port query here: https://github.com/openucx/ucx/pull/3570/files#diff-7cf03c5fefa318dcaeaf6bf7312d1911R44

Is there a simpler way of getting this?

Reply:
this may be not accurate since port=0 can be used when setting self->sockaddr.
maybe something like this?
```
if (sockcm_iface->is_server) {
    struct sockaddr_in sin;
    socklen_t len = sizeof(sin);
    getsockname(sockcm_iface->listen_fd, (struct sockaddr *)&sin, &len);
    iface_attr->listen_port = ntohs(sin.sin_port);
}
```